client/js/widgets/pile.js

```diff
@@ -130,6 +130,14 @@ class Pile extends Widget {
       await this.parent.dispenseCard(child);
   }
 
+  async onPropertyChange(property, oldValue, newValue) {
+    if(property == 'owner') {
```

Comment:
Perhaps we should consider fixing additional widget issues by passing through more property changes.